[BUG] Detection Rule execution authorization model not outlined in security docs (#889)
elastic/security-docs: docs/detections/rules-ui-create.asciidoc

==============

[role="screenshot"]
image::images/all-rules.png[Shows the Rules page]

[float]
[[alerting-authorization-model]]
=== Authorization

> Rules, including all background detection and the actions they generate are authorized using an {kib-ref}/api-keys.html[API key] associated with the last user to edit the rule. Upon creating or modifying a rule, an API key is generated for that user, capturing a snapshot of their privileges at that moment in time. The API key is then used to run all background tasks associated with the rule including detection checks and executing actions.

Reviewer: ```suggestion
Rules, including all background detection and the actions they generate, are authorized using an {kibana-ref}/api-keys.html[API key] associated with the last user to edit the rule. Upon creating or modifying a rule, an API key is generated for that user, capturing a snapshot of their privileges. The API key is then used to run all background tasks associated with the rule, including detection checks and executing actions.
```
